src/Utils/utils.ts

```diff
@@ -101,8 +101,8 @@ export const formatTime = (date: DateLike, format = TIME_FORMAT) =>
 
 export const relativeDate = (date: DateLike, withoutSuffix = false) => {
   const obj = dayjs(date);
-  return `${obj.fromNow(withoutSuffix)} ${
-    withoutSuffix && "ago"
+  return `${obj.fromNow(withoutSuffix)}${
+    withoutSuffix ? " ago " : ""
```

Comment:
```suggestion
    withoutSuffix ? " ago" : ""
```

Reply:
Change LGTM

Reply:
@khavinshankar we did not commit this change before merging 😅